src/Pages/Liked-Videos/likeVideo.js

```diff
@@ -23,7 +25,18 @@ const LikedVideo = () => {
             {likedVideo &&
               likedVideo.map((video) => (
                 <div class="container" key={video._id}>
-                  <figure class="menu-card">
+                  <figure
+                    class="menu-card"
+                    style={{
+                      color: theme === "light" ? "black" : "#fff",
+                      backgroundColor:
+                        theme === "light" ? "#e0e0eb" : "#191921",
+                      boxShadow:
+                        theme === "light"
+                          ? "rgba(0, 0, 0, 0.4) 0px 2px 4px,rgba(0, 0, 0, 0.3) 0px 7px 13px -3px, rgba(0, 0, 0, 0.2) 0px -3px 0px inset"
+                          : "rgba(179, 179, 0) 5px 5px",
+                    }}
```

Comment:
A suggestion: Insteading of passing it everywhere, this can be made into class and used everywhere.